src/partials/modal.html

```diff
@@ -0,0 +1,158 @@
+<div>
+  <div>
+    <h2>Chocolate icecream</h2>
+    <div>
+      <img src="../images/Ellipse67.jpg" alt="icream cone" />
+    </div>
+    <div>
+      <ul>
+        <li><span>320</span>kcal</li>
+        <li><span>4g</span>Protein</li>
+        <li><span>8,2g</span>Fats</li>
+        <li><span>40,4g</span>Carbs</li>
+      </ul>
+      <p>Ingredients</p>
+      <ul>
+        <li>Milk<span>100ml</span></li>
+        <li>Belgian chocolate<span>20g</span></li>
+        <li>Cream<span></span>150 ml</li>
+        <li>Cocoa powder<span>30g</span></li>
+        <li>Sugar<span></span>40g</li>
+        <li>Vanilla sugar<span>10g</span></li>
+      </ul>
+      <form>
+        <div>
+          <input
+            type="text"
+            name="name"
+            id="products-name"
+            placeholder=" "
+            required
+            minlength="2"
+          /><label for="name">Name</label>
+        </div>
+        <div>
+          <input
+            type="tel"
+            name="tel"
+            id="products-tel"
+            placeholder=" "
+            required
+            minlength="9"
+          /><label for="tel">Phone</label>
+        </div>
+        <div>
+          <textarea name="modal-form__textarea" id="products-comment" placeholder=" "></textarea
+          ><label for="comment">Comment</label>
+        </div>
+        <button type="submit">Submit</button>
+      </form>
+    </div>
+  </div>
+</div>
+<div>
+  <div>
+    <h2>Buy now</h2>
+    <form>
+      <div><input type="checkbox" name="buy" value="ice cream" /><label>ice cream</label></div>
+      <div><input type="checkbox" name="buy" value="ice coffee" /><label>ice coffee</label></div>
+      <div><input type="checkbox" name="buy" value="milkshake" /><label>milkshake</label></div>
+      <div>
+        <input
+          type="text"
+          name="name"
+          id="products-name"
+          placeholder=" "
+          required
+          minlength="2"
+        /><label for="name">Name</label>
+      </div>
+      <div>
+        <input
+          type="tel"
+          name="tel"
+          id="products-tel"
+          placeholder=" "
+          required
+          minlength="9"
```

Comment:
trzeba tutaj też dodać jakiś pattern ( jest w wymogach projektu ) i dodaj to do modalu buy now i franchaise modal
Jeśli mógł bym też prosić żeby każdy modal był po kolei tak jak na figmie i oddzielony jakimś komentarzem typu Modal Buy now. Będzie to moim zdaniem czytelniejsze.